test/ts-scenario/steps/channel_operations.ts

```diff
@@ -86,6 +95,7 @@ function validateObjectKeyMatch(expected: any, actual: any, compareType: 'includ
 				if (Object.prototype.hasOwnProperty.call(actual, key)) {
 					// recursive call to scan property
 					BaseUtils.logMsg(`->Recursively checking response key ${key}`);
+					// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Not sure why this is necessary

Reply:
Unable to fix .

Reply:
throwing validateObjectKeyMatch cannot be read . Need to do a workaround